state-transition/core/state_processor_withdrawals.go

```diff
@@ -101,6 +101,7 @@ func (sp *StateProcessor) processWithdrawals(
 			return getErr
 		}
 		updatedWithdrawals := ppWithdrawals[processedPartialWithdrawalsCount:]
+		sp.metrics.gaugePendingPartialWithdrawalsCount(len(updatedWithdrawals))
```

Comment:
nit: we can use `processedPartialWithdrawalsCount` instead of `len(updatedWithdrawals)`